apps/prairielearn/src/components/NavbarContext.html.ts

```diff
@@ -421,12 +14,23 @@ export function ContextNavigation({
   navSubPage: NavSubPage;
 }) {
   if (!navPage) return '';
-  const navPageTabs = navPagesTabs[navPage];
-  if (!navPageTabs) return '';
+
+  const showSideNav = resLocals.has_enhanced_navigation && resLocals.course !== undefined;
+  const navPagesTabs = getNavPageTabs(resLocals.has_enhanced_navigation);
+  const navPageTabs = navPagesTabs[navPage] ?? []; // Some navPages do not have tabs
 
   return html`
     <nav>
-      <ul class="nav nav-tabs pl-nav-tabs-bar pt-2 px-3 bg-light">
+      <ul
+        class="
+          nav 
+          nav-tabs 
+          pl-nav-tabs-bar 
+          ${showSideNav ? 'pt-3' : 'pt-2'} 
```

Comment:
Nit, IMO we should retain the smaller vertical spacing to avoid excessive use of whitespace.

```suggestion
          pt-2
```

Reply:
I agree with removing this since the removal reduces whitespace and simplifies the code a bit. Here's what pages with `ContextNavigation` look like with `pt-2` and the side nav:

![image](https://github.com/user-attachments/assets/3cdbe557-6bde-49e2-9afc-a75a7f4a7415)